Package.swift

```diff
@@ -2,6 +2,48 @@
 
 import PackageDescription
 
+let iOSPlatforms: [Platform] = [.iOS, .visionOS, .macCatalyst]
+let macOSPlatforms: [Platform] = [.macOS]
+
+let targets: [Target] = [
+    .target(
+        name: "FluentUI",
+        dependencies: [
+            .targetItem(name: "FluentUI_ios", condition: .when(platforms: iOSPlatforms)),
+            .targetItem(name: "FluentUI_macos", condition: .when(platforms: macOSPlatforms))
+        ],
+        path: "Sources/FluentUI"
+    ),
+    .target(
+        name: "FluentUI_ios",
+        path: "Sources/FluentUI_iOS",
+        resources: [
+            .copy("Resources/Version.plist")
```

Comment:
do we still need to exclude all the xcfilelists?

Reply:
Surprisingly no! I'm trying to track down the specific version, but at some point SPM started automatically ignoring them when bundling assets.

Reply:
ooh sweet!